cloud/cloudstack/cs_loadbalancer_rule.py

```diff
@@ -0,0 +1,283 @@
+    description:
+      - The public port from where the network traffic will be load balanced from
+      - Required when using C(state=present).
+    required: true
+    default: null
+  public_ip:
+    description:
+      - Public ip address from where the network traffic will be load balanced from
+    required: false
+    default: null
+  open_firewall:
+    description:
+      - Whether the firewall rule for public port should be created, while creating the new rule.
+      - Use M(cs_firewall) for managing firewall rules.
+    required: false
+    default: false
+  cidr_list:
+    description:
+      - The cidr list to forward traffic from
+    required: false
+    default: null
+  project:
+    description:
+      - Name of the project the firewall rule is related to.
+    required: false
+    default: null
+  state:
+    description:
+      - State of the instance.
+    required: true
+    default: 'present'
+    choices: [ 'present', 'absent' ]
+  domain:
+    description:
+      - Domain the rule is related to.
+    required: false
+    default: null
+  account:
+    description:
+      - Account the rule is related to.
+    required: false
+    default: null
+  zone:
+    description:
+      - Name of the zone in which the rule shoud be created.
+      - If not set, default zone is used.
+    required: false
+    default: null
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+# Create a load balancer rule
+- local_action:
+    module: cs_loadbalancer_rule
+    name: balance_http
+    public_ip: 1.2.3.4
+    algorithm: leastconn
+    public_port: 80
+    private_port: 8080
+
+# Delete a load balancer rule
+- local_action:
+    module: cs_loadbalancer_rule
+    name: balance_http
+    state: absent
+'''
+
+RETURN = '''
+---
+id:
+  description: UUID of the rule.
+  returned: success
+  type: string
+  sample: a6f7a5fc-43f8-11e5-a151-feff819cdc9f
+zone:
+  description: Name of zone the rule is related to.
+  returned: success
+  type: string
+  sample: ch-gva-2
+project:
+  description: Name of project the rule is related to.
+  returned: success
+  type: string
+  sample: Production
+account:
+  description: Account the rule is related to.
+  returned: success
+  type: string
+  sample: example account
+domain:
+  description: Domain the rule is related to.
+  returned: success
+  type: string
+  sample: example domain
+'''
+
+
+try:
+    from cs import CloudStack, CloudStackException, read_config
+    has_lib_cs = True
+except ImportError:
+    has_lib_cs = False
+
+# import cloudstack common
+from ansible.module_utils.cloudstack import *
+
+
+class AnsibleCloudStackLBRule(AnsibleCloudStack):
+
+    def __init__(self, module):
+        super(AnsibleCloudStackLBRule, self).__init__(module)
+        self.returns = {
+            'publicip': 'public_ip',
+            'publicport': 'public_port',
+            'privateport': 'private_port',
+            'algorithm': 'algorithm',
+            'cidrlist': 'cidrlist',
```

Comment:
in other module I use just `cidr` for `clidrlist` e.g. in `cs_firewall` https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/cloudstack/cs_firewall.py#L222